.github/workflows/publish.js.yml

```diff
@@ -25,8 +37,36 @@ jobs:
       name: Run build
     - run: npm run test
       name: Run test
+
+    # building WDA packages
+    - name: Build iOS
+      run: |
+        xcodebuild clean build-for-testing -project WebDriverAgent.xcodeproj -derivedDataPath appium_wda_ios -scheme WebDriverAgentRunner -destination generic/platform=iOS CODE_SIGNING_ALLOWED=NO
```

Comment:
long xcodebuild commands could be split into more lines

Reply:
consider moving hardcoded duplicated strings, like `appium_wda_ios` or `WebDriverAgentRunner-Runner.app` into env variables